Makefile

```diff
@@ -48,38 +56,41 @@ clean-dockerdemo:
 ssh:
 	@vagrant ssh netplugin-node1 || echo 'Please run "make demo"'
 
-unit-test: build
-	CONTIV_HOST_GOPATH=$(GOPATH) CONTIV_HOST_GOBIN=$(HOST_GOBIN) \
-					   CONTIV_HOST_GOROOT=$(HOST_GOROOT) ./scripts/unittests -vagrant
+unit-test: stop clean build
```

Comment:
IIRC from discussion in #130 (here: https://github.com/contiv/netplugin/pull/130#discussion_r36670085) the need for `clean` arises due to mismatching go object files when go environment differs between host and vm, right?

doing a `clean` increases build+test time and also need to add it for almost every test target. I am thinking if we should consider keeping the `CONTIV_HOST_GOBIN` and `CONTIV_HOST_GOROOT` vars?